playwright.config.ts

```diff
@@ -1,29 +1,28 @@
-import type { PlaywrightTestConfig } from '@playwright/test';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import { devices } from '@playwright/test';
+import { defineConfig, devices } from '@playwright/test';
 
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
-const config: PlaywrightTestConfig = {
+export default defineConfig({
```

Comment:
Is there a way to have testing coverage report setup as well? This is important to know how much of the codebase are we covering with integration tests and for future PRs.